README.md

```diff
@@ -1,330 +1,259 @@
-        <a href="https://twitter.com/hamilton_os" target="_blank"><img src="https://img.shields.io/twitter/url/http/shields.io.svg?style=social" alt="Twitter"/></a>
-        <br/>
-    <a href="https://www.python.org/downloads/" target="_blank"><img src="https://img.shields.io/badge/python-3.8|%203.9|%203.10|%203.11|%203.12-brightgreen.svg" alt="Python supported"/></a>
-    <a href="https://pypi.org/project/sf-hamilton/" target="_blank"><img src="https://badge.fury.io/py/sf-hamilton.svg" alt="PyPi Version"/></a>
-    <a href="https://pepy.tech/project/sf-hamilton" target="_blank"><img src="https://pepy.tech/badge/sf-hamilton" alt="Total Downloads"/></a>
-    <br/>
-    <a href="https://anaconda.org/hamilton-opensource/sf-hamilton"> <img src="https://anaconda.org/hamilton-opensource/sf-hamilton/badges/version.svg" /> </a>
-    <a href="https://anaconda.org/hamilton-opensource/sf-hamilton"> <img src="https://anaconda.org/hamilton-opensource/sf-hamilton/badges/platforms.svg" /> </a>
+   <h1><img src="https://github.com/DAGWorks-Inc/hamilton/assets/2328071/feb6abaa-b6d5-4271-a320-0ae4a18d8aa7" width="50"/> Hamilton — portable & expressive <br> data transformations DAG</h1>
+   <a href='https://hamilton.dagworks.io/en/latest/?badge=latest'>
+      <img src='https://readthedocs.org/projects/hamilton/badge/?version=latest' alt='Documentation Status' />
+   </a><a href="https://www.python.org/downloads/" target="_blank">
+      <img src="https://img.shields.io/badge/python-3.8|%203.9|%203.10|%203.11|%203.12-blue.svg" alt="Python supported"/>
+   </a>
+   <a href="https://pypi.org/project/sf-hamilton/" target="_blank">
+      <img src="https://badge.fury.io/py/sf-hamilton.svg" alt="PyPi Version"/>
+   </a>
+   <a href="https://pepy.tech/project/sf-hamilton" target="_blank">
+      <img src="https://pepy.tech/badge/sf-hamilton" alt="Total Downloads"/>
+   </a>
+
+   <a target="_blank" href="https://linkedin.com/showcase/dagster" style="background:none">
+      <img src="https://img.shields.io/badge/DAGWorks-Follow-purple.svg?logo=linkedin" />
+   </a>
+   <a href="https://join.slack.com/t/hamilton-opensource/shared_invite/zt-1bjs72asx-wcUTgH7q7QX1igiQ5bbdcg" target="_blank">
+      <img src="https://img.shields.io/badge/Hamilton-Join-purple.svg?logo=slack" alt="Hamilton Slack"/>
+   </a>
+   <a href="https://twitter.com/hamilton_os" target="_blank">
+      <img src="https://img.shields.io/badge/DAGWorks-Follow-purple.svg?logo=X"></src>
+   </a>
 </div>
+<br></br>
 
-# Hamilton
+Hamilton is a general purpose Python library to author data transformations as a directed acyclic graph (DAG). It enables users to write ETL pipelines, ML workflows, LLM applications, RAG systems, business analytics dashboards, webscrapers, and more! The [Hamilton UI](https://hamilton.dagworks.io/en/latest/concepts/ui) allows to automatically visualize, catalog, and monitor execution.
```

Comment:
"general purpose python library to author" doesn't sell value. While we want it to be matter-of-fact, it should reflect/reiterate the tagline.